config/F16.config

```diff
@@ -4,6 +4,17 @@ process {
         memory = 1.GB
         }
 
+    withName: preprocess_BAM_GRIDSS {
+        cpus = 1
+        memory = 26.GB
```

Comment:
looks like only 6GB is utilized. Testing a run with 4 CPUs and 10GB of memory. Will update this thread once done.

Reply:
Test successful with the recommended 4CPUs by GRIDSS.
/hot/software/pipeline/pipeline-call-sSV/Nextflow/development/unreleased/mmootor-add-GRIDSS/preprocess-4cpu-10GBMem.log